alt/asyncapi/src/commands/context/add.ts

```diff
@@ -0,0 +1,47 @@
+import {Command, flags} from '@oclif/command'
```

Comment:
Oclif supports nested files and folders inside the `src/commands` folder. You want a subcommand? Nest a file inside a folder and you get it 💯